.github/workflows/test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
Just to check: do these need to be strings because the yml template lops the trailing zero off 3.10 (treats as float)? That is good to know.

Reply:
Yes! See commit `f95bbd6`, it attempted to test in Python 3.1